test-data/unit/check-bound.test

```diff
@@ -215,3 +215,11 @@ if int():
     b = 'a' # E: Incompatible types in assignment (expression has type "str", variable has type "int")
 twice(a) # E: Value of type variable "T" of "twice" cannot be "int"
 [builtins fixtures/args.pyi]
+
+
+[case testIterableBoundUnpacking]
+from typing import Tuple, TypeVar
+TupleT = TypeVar("TupleT", bound=Tuple[int, ...])
+def f(t: TupleT) -> None:
+    a, *b = t
```

Comment:
Let's add `reveal_type(a)` and `reveal_type(b)` here. This would catch some unexpected types.

Reply:
Done!